support/mesos-tidy/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:xenial
+FROM --platform=linux/amd64 ubuntu:xenial
```

Comment:
> consistent would be to drop the modification here and instead document in the readme docker build --platform=linux/amd64

ah yeah true, good point, I could move the platform to the build command itself rather than burn it into the Dockerfile

> such a setup would still only work if the host is actually amd64 (and e.g., fail on arm setups like newer macs unless the Docker VM emulates amd64 and does not use the default arm64 architecture

hm.. I am on an M1 mac, and encountered the following error that prompted me to add the platform which fixed the error:

https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc

Then when running I would get the following, which prompted me to add the `--platform` arg on the run command as well:

```
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

By the way, are you able to add me as a collaborator on the mesos organization in Docker hub? I don't seem to have access.